applications/nrf_desktop/Kconfig.ble

```diff
@@ -37,44 +44,125 @@ config BT_ID_UNPAIR_MATCHING_BONDS
 
 config BT_DATA_LEN_UPDATE
 	default n
+	help
+	  nRF Desktop peripherals disable the Data Length Exchange (DLE)
+	  procedure as it expects this procedure to be initiated by the
+	  the central.
 
 config BT_AUTO_PHY_UPDATE
 	default n
+	help
+	  nRF Desktop peripherals disable the automatic PHY Update procedure
+	  as it expects this procedure to be initiated by the central.
 
 config BT_PERIPHERAL_PREF_MIN_INT
 	default 6
+	help
+	  nRF Desktop peripherals use the shortest available connection
+	  interval to lower the latency of HID reports.
 
 config BT_PERIPHERAL_PREF_MAX_INT
 	default 6
+	help
+	  nRF Desktop peripherals use the shortest available connection
+	  interval to lower the latency of HID reports.
 
 config BT_PERIPHERAL_PREF_LATENCY
 	default 99
+	help
+	  nRF Desktop peripherals use the non-zero value for the Bluetooth
+	  latency to lower the battery energy consumption.
 
 config BT_PERIPHERAL_PREF_TIMEOUT
 	default 400
+	help
+	  nRF Desktop peripherals use the moderate value for the supervision
+	  timeout parameter as they do not have to be fast in the disconnect
+	  detection.
 
 config BT_CONN_PARAM_UPDATE_TIMEOUT
 	default 1000
+	help
+	  nRF Desktop peripherals update connection parameters at the beginning
+	  of the connection session to lower the packet latency before the HID
+	  report exchange.
 
 endif
 
 config DESKTOP_BT_CENTRAL
 	bool
 	default y if DESKTOP_ROLE_HID_DONGLE
 	select BT_CENTRAL
+	help
+	  nRF Desktop dongles are configured to perform the Bluetooth Central
```

Comment:
`Only nRF Desktop dongles...`

As it is worded now it would seem that it is selected for dongles because it is more resource intensive, which is not the case - it's there because it's needed.

Reply:
Hmm... I have changed it since there are only two roles and hid peripherals are not using the Central role.

Reply:
``` it's there because it's needed```
I don't think this is a good explanation

As far as I understand, HID peripherals use Bluetooth Peripheral role because they are smaller devices and have more limited resources in contrast to the dongle devices which perform the Bluetooth Central role. From my perspective, it is the only reason why this choice was made in the nRF Desktop application. What is the other reason to keep these roles as they are if we control both ends of Bluetooth connectivity (Central and Peripheral sides)?

Reply:
this is great explanation: https://github.com/nrfconnect/sdk-nrf/pull/9435#discussion_r1031327168